src/test/java/igym/services/GymServiceTest.java

```diff
@@ -209,4 +210,56 @@ void testFindInactiveGym() {
                 () -> gymService.findById(gymId));
         assertEquals("Gym with id " + gymId + " not found", exception.getMessage());
     }
+
+    @Test
+    @DisplayName("should return gyms when user exists and gyms are active")
+    void testFindGymsByUserId() {
+        UUID userId = UUID.randomUUID();
+        User user = new User("Test User");
+        Gym gym1 = new Gym("Gym 1");
+        Gym gym2 = new Gym("Gym 2");
+        gym1.setStatus(Status.active);
+        gym2.setStatus(Status.active);
```

Comment:
```suggestion
        Gym gym1 = new Gym("Gym 1");
        Gym gym2 = new Gym("Gym 2");
        Gym gym3 = new Gym("Gym 3");
        gym1.setStatus(Status.active);
        gym2.setStatus(Status.active);
        gym3.setStatus(Status.inactive);
```

Despite the DB query is mocked, I suggest adding an inactive gym to "test" that inactive gyms aren't being returned.
BTW, soon we will need to review these mocks. We may be hiding some bugs because of it 🐛 👻